3-platform/shopping-cart/app/src/main/java/com/_6core/platform/shopping/cart/app/validator/Validator.java

```diff
@@ -0,0 +1,5 @@
+package com._6core.platform.shopping.cart.app.validator;
+
+public interface Validator<T> {
+  Boolean validate(final T entity);
```

Comment:
Dont use standard java types (String, Integer, Long) as return/input types. 
Create your own class even if it has only one field. 

Entity is input param? Entity from database? 
```suggestion
  ??? validate(final T entity??);
```